### PR TITLE
Separate chain calls

### PR DESCRIPTION
**Pull request by sankaest**
| Q             | A
| ------------- | ---
| Readability fix | according to https://github.com/yiisoft/docs/issues/155, chain calls must be put on separate lines

### CODE REVIEW

File: src/Redis/LuaScriptBuilder.php [hunk 1]
```diff
@@ -39,7 +39,11 @@ final class LuaScriptBuilder
      */
     public function buildAll(ActiveQuery $query): string
     {
-        $key = $this->quoteValue($query->getARInstance()->keyPrefix() . ':a:');
+        $key = $this->quoteValue(
+            $query
+                ->getARInstance()
+                ->keyPrefix() .
+            ':a:');
```
Comment (vjik) [inline src/Redis/LuaScriptBuilder.php:46]:
```suggestion
        $key = $this->quoteValue(
            $query
                ->getARInstance()
                ->keyPrefix() .
            ':a:'
        );
```

File: tests/Pgsql/ActiveRecordTest.php [hunk 2]
```diff
@@ -121,23 +127,73 @@ public function testBooleanValues(): void
...
-        $this->assertEquals(1, $boolARQuery->where(['bool_col' => false])->count('*'));
-        $this->assertEquals(2, $boolARQuery->where(['bool_col' => [true, false]])->count('*'));
-
-        $this->assertEquals(1, $boolARQuery->where('bool_col = :bool_col', ['bool_col' => true])->count('*'));
-        $this->assertEquals(1, $boolARQuery->where('bool_col = :bool_col', ['bool_col' => false])->count('*'));
+        $this->assertTrue(
+            $boolARQuery
+            ->where(['bool_col' => true])
+            ->one()->bool_col
+        );
```
Comment (vjik) [inline tests/Pgsql/ActiveRecordTest.php:140]:
```suggestion
        $this->assertTrue(
            $boolARQuery
                ->where(['bool_col' => true])
                ->one()->bool_col
        );
```

File: tests/Pgsql/ActiveRecordTest.php [hunk 3]
```diff
@@ -121,23 +127,73 @@ public function testBooleanValues(): void
...
+        $this->assertFalse(
+            $boolARQuery
+            ->where(['bool_col' => false])
+            ->one()->bool_col
+        );
```
Comment (vjik) [inline tests/Pgsql/ActiveRecordTest.php:145]:
```suggestion
        $this->assertFalse(
            $boolARQuery
                ->where(['bool_col' => false])
                ->one()->bool_col
        );
```

File: tests/ActiveQueryTest.php [hunk 4]
```diff
@@ -1955,27 +2288,42 @@ public function testInverseOfDynamic(): void
...
+            ->getOrders2()
+            ->with('customer2')
+            ->one();
         $this->assertSame($customer, $orders2->customer2);
 
         /** request the inverseOf relation as array */
-        $orders2 = $customer->getOrders2()->asArray()->all();
+        $orders2 = $customer->getOrders2()
+            ->asArray()
+            ->all();
```
Comment (vjik) [inline tests/ActiveQueryTest.php:2320]:
```suggestion
        $orders2 = $customer
            ->getOrders2()
            ->asArray()
            ->all();
```

**Review by vjik — CHANGES_REQUESTED** [review]
Why added `tests/Data/Runtime/yiitest.sq3` ? Seems, it needs to be deleted.

**Comment by vjik** [comment]
@sankaest what about `tests/Data/Runtime/yiitest.sq3` ?

**Comment by sankaest** [comment]
@vjik I don't know where it came from :( . I have not added it, but maybe somehow accidentally, as I still quite new to git/gitHub and still learning (sometimes by deleting whole changes I made, you know - checkout/master, brrrr :) ).
I delete this file.

**Comment by vjik** [comment]
> @vjik I don't know where it came from :( . I have not added it, but maybe somehow accidentally, as I still quite new to git/gitHub and still learning (sometimes by deleting whole changes I made, you know - checkout/master, brrrr :) ).
> I apologize and I delete this file.

It's OK. You gain experience!

**Review by vjik — APPROVED** [review]

**Pull request merged**